runtime/appruntime/metrics/system/system.go

```diff
@@ -0,0 +1,19 @@
+package system
+
+import (
+	"runtime"
+)
+
+type SysMetrics struct {
+	MemoryUsageBytes uint64
+	NumGoRoutines    int
```

Comment:
Why num goroutines and not CPU usage?

Reply:
I haven't found a way to get the current CPU usage.